packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode, useState } from 'react'
+
+import { Divider, Item, List } from './styled'
+
+interface AccordionItemProps {
+  children?: ReactNode
+  className?: string
+}
+/* istanbul ignore next */
+export function useAccordionController() {
+  const [activeItem, setActiveItem] = useState<number>()
```

Comment:
Nit: It might be better to be explicit here with `null`, both for the value and the type.

`const [activeItem, setActiveItem] = useState<number | null>(null)`